StreamChatClient_v3/Controllers/ChannelListController.swift

```diff
@@ -30,7 +30,7 @@ public class ChannelListController<ExtraData: ExtraDataTypes>: Controller, NSFet
         try! fetchResultsController.performFetch()
         fetchResultsController.delegate = self
         
-        channels = fetchResultsController.fetchedObjects!.map(ChannelModel<ExtraData>.init)
+        channels = fetchResultsController.fetchedObjects!.map { ChannelModel<ExtraData>.create(fromDTO: $0) }
```

Comment:
Doesn't the old version work anymore?
`.map(ChannelModel<ExtraData>.create)`

Reply:
It does. This is a leftover. Thanks

Reply:
fixed in f62d5db